src/plugin/file/kml/kmlparser.js

```diff
@@ -841,6 +849,55 @@ plugin.file.kml.KMLParser.prototype.createTreeNode_ = function(el, opt_parent) {
 };
 
 
+/**
+ * Examine styles that are unsupported in OpenLayers KML styles.
+ * @param {Node} node Node.
+ * @private
+ */
+plugin.file.kml.KMLParser.prototype.examineStyles_ = function(node) {
+  var n;
+  for (n = node.firstElementChild; n; n = n.nextElementSibling) {
+    if (n.localName == 'BalloonStyle') {
+      var properties = ol.xml.pushParseAndPop({}, plugin.file.kml.BALLOON_PROPERTY_PARSERS, n, []);
+      if (properties) {
+        var id = node.id || node.getAttribute('id');
+        this.otherStyleMap[id] = properties;
+      }
+    }
+  }
+};
+
+
+/**
+ * Read the KML balloon style.
+ * @param {ol.Feature} feature The feature
+ * @private
+ */
```

Comment:
Add `@suppress {accessControls}` to fix one of the build failures

Reply:
Ah okay I was wondering what the best way to get past that build error was, thanks for that!!